pkgs/development/python-modules/pushover/default.nix

```diff
@@ -12,7 +12,7 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ requests ];
 
-  # checks crash
+  # there are no tests
```

Comment:
@Mic92 That's not true: https://github.com/laprice/pushover/blob/master/test.py

Reply:
fixed: c3535901c3b64e244604be89b144a0e3330525f9

Reply:
@Mic92 Thanks!